corehq/ex-submodules/soil/tests/test_download_base.py

```diff
@@ -30,6 +30,7 @@ def test_expose_blob_download(self):
 
         expose_blob_download(
             self.identifier,
+            expiry=60,
```

Comment:
Should this be `expiry=60 * 60`? In other files it looks like `expose_blob_download` is called with a number of seconds.

Reply:
This is just a test case so 60s seems plenty